rott/rt_datadir.c

```diff
@@ -255,7 +275,34 @@ static void AddStorefrontDirs(void)
 		prefix = pwd->pw_dir;
 	}
 #else
-	const char prefix[] = "C:";
+	char *prefix = getenv("USERPROFILE");
+	char prefix1[1024];
+
+	if (!prefix)
+	{
+		char *homedrive = getenv("HOMEDRIVE");
+		char *homepath = getenv("HOMEPATH");
+
+		if (homedrive && homepath)
+		{
+			M_snprintf(prefix1, sizeof(prefix1), "%s%s", homedrive, homepath);
```

Comment:
You could just use `M_StringJoin()` here and call `free(prefix1)` later instead of allocating another 1kB string buffer. Also, no need to cast a `char` pointer to a `char` pointer.